server/models/Request.js

```diff
@@ -0,0 +1,50 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+const timeSlot = new Schema({ start: Date, end: Date });
+
+const requestSchema = new Schema({
+  ownerId: {
+    type: Schema.Types.ObjectId,
+    ref: "user",
+    required: true,
+  },
+  sitterId: {
+    type: Schema.Types.ObjectId,
+    ref: "user",
+    required: true,
+  },
+  duration: {
+    type: timeSlot,
+    required: true,
+  },
+  accepted: {
```

Comment:
Optional: You can use an enum like object as opposed to having multiple fields for accepted , viewed and completed

Reply:
Great Idea!